src/video/ogc/SDL_ogckeyboard.c

```diff
@@ -0,0 +1,59 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2023 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+#include "SDL_ogckeyboard.h"
+#include "../../events/SDL_keyboard_c.h"
+
+#if defined(SDL_VIDEO_DRIVER_OGC) && defined(__wii__)
```

Comment:
I think you can just do
```suggestion
#ifdef __wii__
```
Because we are not using the SDL_VIDEO_DRIVER_OGC include guard in any other file of this directory, unless I'm mistaken.

Reply:
SDL_ogcmouse.c uses this so I pulled it from there. It can be removed from both because CMake handles this AFAIK.